ScarletEngine/Sources/Engine/Private/Engine.cpp

```diff
@@ -48,15 +46,16 @@ namespace ScarletEngine
 
 		std::chrono::high_resolution_clock Clock;
 
-		using seconds = std::chrono::duration<double, std::ratio<1>>;
+		using ms = std::chrono::duration<double, std::ratio<1>>;
 		auto LastTime = Clock.now();
 		double Lag = 0.0;
 
 		bIsRunning = true;
 		while (bIsRunning)
 		{
 			FrameMark
-			const double DeltaTime = std::chrono::duration_cast<seconds>(Clock.now() - LastTime).count();
+
+			const double DeltaTime = std::chrono::duration_cast<ms>(Clock.now() - LastTime).count();
```

Comment:
Please revert these changes. This is inaccurate. `ms` in this instance is actually a ratio representing seconds not milliseconds.